test/test_nwb_model_interpreter.py

```diff
@@ -1,6 +1,6 @@
 import traceback
 import unittest
```

Comment:
This is yet to be moved to pytest. @rodriguez-facundo do you prefer to do that now or maybe we can leave it for another time

Reply:
I will do it now

Reply:
done (I guess)